contrib/init/bitcoind.init

```diff
@@ -26,7 +26,7 @@ lockfile=${BITCOIND_LOCKFILE-/var/lock/subsys/bitcoind}
 bitcoind=${BITCOIND_BIN-/usr/bin/bitcoind}
 
 # bitcoind opts default to -disablewallet, override with BITCOIND_OPTS
-bitcoind_opts=${BITCOIND_OPTS--disablewallet}
+bitcoind_opts=${BITCOIND_OPTS--conf=/.bitcoin/bitcoin.conf -datadir=/.bitcoin -daemon -disablewallet}
```

Comment:
Note that it is possible to specify more options by setting `BITCOIND_OPTS`.

Instead of adding these options (whose values don't make sense) you could improve the comment above by mentioning these notable options?

Reply:
@promag @mysticaltech yeah, if you want to change the -conf or -datadir setting, then using BITCOIND_OPTS (probably by setting it in /etc/sysconfig/bitcoind) is the best way.

As for the --daemon option, I think that's correct for modern bitcoin exes. 
I'll try to give this a test on centos6 in the next couple of days.

Reply:
Our goal here is to have a working default bitcoind.init file, so the only option not necessary here is `-disablewallet`, maybe we could leave this one for the user to setup in `/etc/sysconfig/bitcoind`, what do you think?